releasenotes/notes/49364.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 49364
+releaseNotes:
+- |
+  **Fixed** an bug where VirtualServices containing hosts that aren't present in the service registry are ignored
```

Comment:
is n't the problem specific to wildcard hosts? We should clarify that in stead of saying the problem is with non registry hosts?